nexus/tests/integration_tests/endpoints.rs

```diff
@@ -257,6 +289,10 @@ pub enum AllowedMethod {
     /// that we define here or uuids that we control in the test suite (e.g.,
     /// the rack and sled uuids).
     GetNonexistent,
+    /// HTTP "GET" method that is not yet implemented
+    ///
+    /// This should be a transient state, used only for stub APIs
+    GetUnimplemented,
```

Comment:
do we anticipate needing this for other methods? e.g. `PostUnimplemented`?

Reply:
I don't think so.  For everything other than `GET`, we only make requests that we expect to fail for authn/authz reasons.  For `GET`, do that, but we also make a request with a privileged user that we expect to succeed.  `GetUnimplemented` changes our expectation about this privileged request -- that it will fail with a 500.

Reply:
I added a comment to this effect on this variant and the previous variant in c4d8e1fbd217a323522893f218a191f47c375f71.